### PR TITLE
Navigation: Reduce side bar nesting for commonly used sections

### PR DESCRIPTION
**Pull request by travier**
nav: Move 'User guides' sub sections at the root

---

nav: Merge bare metal & lives pages into Provisioning section

### CODE REVIEW

**Comment by travier** [comment]
Before:
Screenshot_20210331_181423

After:
Screenshot_20210331_181544

**Comment by travier** [comment]
I frequently navigate across those sections and find it unnecessarily redundant to have to open the User Guides one every time I look at the docs.

**Review by jlebon — APPROVED** [review]
This makes a lot of sense to me!

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,8 +1,9 @@
 * xref:faq.adoc[FAQ]
 * xref:getting-started.adoc[Getting Started]
-* xref:bare-metal.adoc[Installing on Bare Metal]
 * Provisioning Machines
 ** xref:stream-metadata.adoc[Stream metadata]
+** xref:bare-metal.adoc[Installing on Bare Metal]
+** xref:live-booting-ipxe.adoc[Live-booting via iPXE (transient system)]
```
Comment (bgilbert) [inline modules/ROOT/nav.adoc:6]: Maybe we could drop `(transient system)` to save pixels, since it's redundant with `Live-booting`.

**Review by bgilbert — APPROVED** [review]
:+1:

**Comment by travier** [comment]
Screenshot_20210402_121601

**Comment by dustymabe** [comment]
Only concern I have is: Will this change break links if people have referenced content from other places (forums, issues, etc)?

**Comment by bgilbert** [comment]
This shouldn't change any page URLs AFAICT.

**Review by lucab — APPROVED** [review]

**Pull request merged**